crud-movie-quickstart/src/test/java/org/acme/MoviesEndpointIT.java

```diff
@@ -0,0 +1,171 @@
+package org.acme;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+
+@QuarkusIntegrationTest
+public class MoviesEndpointIT {
```

Comment:
@cescoffier why is this class duplicating `MoviesEndpointTest`, shouldn't it just be:

```suggestion
public class MoviesEndpointIT extends MoviesEndpointTest {
```